GoogleForCreators/web-stories-wp
# Update Moveable to fix right-click handling + resizing

## Summary

Prevents dragging when the user is either using right-click or using Ctrl when clicking.


## Relevant Technical Choices
Right-click without using an actual mouse is different from using it on a laptop without a mouse.
This PR prevents dragging when Ctrl is down.


## To-do

-



## User-facing changes

When using right-click or Ctrl + Left-click for opening the context menu, the drag event is not started.


## Testing Instructions
I
1. Add a shape
2. Open the context menu on the shape by using Ctrl + left click
3. Close the context menu by clicking somewhere outside
4. Move the mouse around
5. Verify that dragging is not happening.

II
1. Add an image
2. Set rotation to 90 degrees
3. Now drag the image to the upper left corner
4. Resize it from the corners
5. Verify the resizing works as expected and that the image doesn't "jump around".



---



Fixes #2087 
Fixes #1011
